docs/src/dbns.md

```diff
@@ -0,0 +1,54 @@
+# Dynamic Bayesian Networks
+
+Part of the conceptual definition of a POMDP or MDP is a dynamic Bayesian network (DBN) that defines which random variables are dependent on each other.
```

Comment:
Link to an external ref for DBNs? It isn't the most palate-friendly topic. Plus DMU uses the term 'decision diagram' more than DBN

Reply:
Yeah, what reference should we use? @mykelk  is DBN the right word to use? We are trying to refer to a graphical model for one step that has nodes s, a, s', o, r (figure 4.1 b in the book). In your vocabulary, what is the difference between a DBN and a "decision diagram". I'm not sure if I've ever heard "decision diagram" from anyone else.

Reply:
Just to put this out there: In my classes people referred to DBNs as pure inference problems in the sense of "generalized HMMs". Whenever there were decision nodes involved, people rather used the "Dynamic Decision Network" (so that's another one).

Reply:
DDN seems the best at the moment. The more generic "Graphical Structure" would also be an object

Reply:
I think a "decision diagram" is something else, it refers to B(inary)DDs and A(lgebraic)DDs.
Dynamic Decision Network is probably the best. I have seen "Influence Diagrams" as well.